server/database/models/TournamentModel.ts

```diff
@@ -32,4 +35,18 @@ TournamentSchema.pre(/^find/, function (next) {
   next();
 });
 
-export default mongoose.model<TournamentWihtoutMS & Document>('Tournament', TournamentSchema);
+interface MatchSideDocument extends Omit<MatchSideWithoutMS, 'team'>, Document {
+  team: TeamDocument;
+}
+export interface MatchDocument extends Omit<MatchWithoutMS, 'id' | 'sideA'| 'sideB'>, Document {
+  sideA: MatchSideDocument
+  sideB: MatchSideDocument
+
+}
```

Comment:
```suggestion
export interface MatchDocument extends Omit<MatchWithoutMS, 'id' | 'sideA' | 'sideB'>, Document {
  sideA: MatchSideDocument
  sideB: MatchSideDocument
}
```
Is Linter not working?

Reply:
Idk, it is.